rpm/qgis.spec.template

```diff
@@ -233,40 +233,13 @@ desktop-file-install \
     --remove-mime-type="application/x-raster-ecw" \
     --remove-mime-type="application/x-raster-mrsid" \
     --dir=%{buildroot}%{_datadir}/applications \
-    debian/qgis.desktop
```

Comment:
All this block can be removed since the `.dekstop` file is now already in the `builddir`, otherwise two icons will appears in menus